web/src/main/java/org/cbioportal/web/StudyViewController.java

```diff
@@ -304,33 +240,6 @@ public List<CopyNumberCountByGene> cacheableFetchCNAGenes(StudyViewFilter interc
         return copyNumberCountByGenes;
     }
 
-    @PreAuthorize("hasPermission(#involvedCancerStudies, 'Collection<CancerStudyId>', T(org.cbioportal.utils.security.AccessLevel).READ)")
-    @RequestMapping(value = "/filtered-samples/fetch", method = RequestMethod.POST,
-        consumes = MediaType.APPLICATION_JSON_VALUE, produces = MediaType.APPLICATION_JSON_VALUE)
-    @ApiOperation("Fetch sample IDs by study view filter")
-    public ResponseEntity<List<Sample>> fetchFilteredSamples(
-        @ApiParam("Whether to negate the study view filters")
-        @RequestParam(defaultValue = "false") Boolean negateFilters,
-        @ApiIgnore // prevent reference to this attribute in the swagger-ui interface
-        @RequestAttribute(required = false, value = "involvedCancerStudies") Collection<String> involvedCancerStudies,
-        @ApiIgnore // prevent reference to this attribute in the swagger-ui interface. this attribute is needed for the @PreAuthorize tag above.
-        @Valid @RequestAttribute(required = false, value = "interceptedStudyViewFilter") StudyViewFilter interceptedStudyViewFilter,
-        @ApiParam(required = true, value = "Study view filter")
-        @Valid @RequestBody(required = false) StudyViewFilter studyViewFilter) {
```

Comment:
Some of these decorators don't seem to be in the columnstorecontrolller endpoint? Should we still copy those over? Or were they redundant?